### ISSUE TITLE
_unmeasured does not gate recall/no-overfetch, so a treatment transport loss is still scored as a behavioural miss

### ISSUE THREAD

**Issue by inth3shadows** [inth3shadows/terse#371]
Split out of #352 rather than folded in, because closing it needs a gate where none exists today.

## What #352 fixed, and what it did not

#352 made `_unmeasured` see dropeval's treatment arm. But `_unmeasured` only ever runs on the **final-accuracy** metric, via `_accuracy_gate` -> `arm_gap`. Recall and no-overfetch call `_form_stats` directly against a **fixed 100% ideal** (`report.py`, in `dropeval_verdict`), so they never pair, never build an `ArmGap`, and have never been gated on transport loss at all.

Consequence, executed on identical loss placed on opposite arms (24+24 rows, `trials=10`, 4 calls lost per row):

```
treatment lost 4/10    directive=BLOCK
   - Worst-case model `m`: retrieve-recall 60% vs ideal (100%) 100% (gap -40%). **FAIL**
   - **final-accuracy: not gated for `m`** — too few calls to compare.
control lost 4/10      directive=NOT_CONCLUDED
   - Worst-case model `m`: retrieve-recall 100% vs ideal (100%) 100% (gap +0%). **PASS**
   - **final-accuracy: not gated for `m`** — too few calls to compare.
```

The rendered treatment report contradicts itself within one screen:

> **Model calls failed** — these rows measure the harness, not the model: `m` 192/960

...then `final-accuracy: not gated ... too few calls to compare`, then `retrieve-recall 60% ... **FAIL**` — computed from the same rows the paragraph above just declared unmeasurable.

## Why this is not urgent

The direction is conservative. A treatment transport loss can only make the verdict *stricter* (BLOCK, never SHIP): `retrieve_ok <= trials - errors` and dropeval emits no `retrieve_trials`, so any loss past `UNMEASURED_FAIL_SHARE` forces recall <= 80%, under the 95% tolerance, and the fixed-ideal metric contributes BLOCK regardless of what the accuracy gate decides.

An exhaustive sweep of 8,640 dropeval fleets (single-model and two-model, base vs #352) found **zero** directive loosenings and zero gained ship authorizations.

So the cost is not a false green. It is that a degraded run is reported as a **behavioural** failure of the drop rule, when what actually happened is that the backend stopped answering. An operator reading `retrieve-recall 60% **FAIL** — keep drop-to-retrieve off` has been told something about the model that the harness did not measure.

## What to decide

The fixed-ideal metrics have no control arm, so `_unmeasured`'s per-arm machinery does not transfer as-is. Options:

1. Gate the fixed-ideal metrics on the treatment arm's transport loss directly (the `<arm>_errors` share #352 added), withholding recall/precision the way accuracy is withheld. Symmetric, but `_form_stats` is called from an AST-enforced allowlist (`test_gap_gate_boundary.py`) — adding a gate means deciding where it lives so a seventh call site cannot skip it.
2. Leave the metrics scored and make the *report* state the contradiction — say plainly that a recall FAIL on a degraded run may be transport, not behaviour. Cheaper, and #352's own framing allows it ("or the report states why it does not"), but it leaves a number in the table that the paragraph above disowns.

## Already pinned

`tests/test_unmeasured_arm_symmetry.py::test_the_run_level_verdict_is_still_asymmetric_and_that_is_recorded_not_fixed` asserts the current asymmetry **and its direction** — a treatment loss may never produce a weaker directive than the same loss on the control. Closing this issue turns that test red, which is intended: it tells whoever does it to delete the test rather than letting the fix land silently.

Related: #352 (the accuracy half), #299 (arm-correlated attrition), #269 (the control arm).

**Comment by inth3shadows** [comment]
**Correction to this issue's "Why this is not urgent" section — the harmless-band claim is false, and it shaped the first fix badly.**

The body argues:

> `retrieve_ok <= trials - errors` and dropeval emits no `retrieve_trials`, so any loss past `UNMEASURED_FAIL_SHARE` forces recall <= 80%, under the 95% tolerance, and the fixed-ideal metric contributes BLOCK regardless

That is true as far as it goes, and it silently invites the reading "so a loss BELOW `UNMEASURED_FAIL_SHARE` is fine". It is not. **The loss share and the tolerance are different numbers**, and the defect lives in the gap between them. Executed on `main`, 10% loss with the model correct on every call that landed:

```
| `m` | 24 | 90% ±0 | 90% ±0 | 90% ±0 | 100% | 90% ±0 | 48/960 |
> **Model calls failed** — these rows measure the harness, not the model: `m` 48/960.
- Worst-case model `m`: retrieve-recall 90% vs ideal (100%) 100% (gap -10% ±0 pts). **FAIL** at 5% tolerance.
```

That is this issue's screenshot with different digits. Tolerance is 5% and every lost call scores a miss, so the defect is live across the whole **(5%, 20%]** band — a range the body implies is safe.

## Why it matters beyond bookkeeping

The body's option 1 says "gate the fixed-ideal metrics on the treatment arm's transport loss directly (the `<arm>_errors` share #352 added)". I implemented exactly that, and adversarial review found it wrong in **both** directions:

- It misses the `(5%, 20%]` band entirely, per above.
- Worse, it withholds **demonstrated** failures. 21% loss with `retrieve_ok` 0 of the 79 calls that LANDED is a mechanism that does not work — the loss bounds recall at 79%, and the observed 0% is 79 points below anything transport can account for. A sweep of 1,440 two-model fleets turned **240 BLOCKs into NOT_CONCLUDED** that way. `NOT_CONCLUDED (2) < BLOCK (3)`, so for the withheld model the exclusion IMPROVES the verdict — the exact shape `UNMEASURED_FAIL_SHARE`'s own comment refuses:

  > Do not add a survival threshold here without first making an exclusion unable to improve a verdict; those are one change, not two.

So a loss-share threshold is the wrong predicate, not merely a mis-tuned one. The question is not *is the loss large* but **could the loss explain this miss**.

## What #379 ships instead

```python
acc, se = _form_stats(kind_rows, "retrieve_ok")
loss = _arm_loss_share(kind_rows, "treatment")
if loss and not passes_tolerance(acc - 1.0) and passes_tolerance(acc + loss - 1.0):
    excluded_by_metric[mech][model] = "unmeasured"
    continue
```

Score first; withhold only a FAIL that crediting every lost call would clear. `acc + loss` is that ceiling, and the two share a denominator. A failure that survives crediting the whole loss is behavioural and gets scored — so an exclusion here can no longer improve any model's verdict, which is the invariant rather than a consequence of where a threshold sits. It is `thin_by_metric`'s shape mirrored: that withholds a PASS too thin to mean anything, this withholds a FAIL too degraded to mean anything.

There is no threshold left, so there is no `>` boundary to get wrong — the old one was unpinned, and mutating it to `>=` survived the entire suite.

Verified by execution, all three:

```
21% loss, 0 of 79 landed correct   ->  BLOCK, gap -1.0        (was withheld)
10% loss, perfect on landed calls  ->  not gated              (was **FAIL**)
no loss, model genuinely failing   ->  BLOCK                  (unchanged)
```

## Two other things review turned up in the same area

- `_exclusion_remedy`'s `"unmeasured"` prose — "Too few calls completed on **BOTH arms** to compare ... a zero means an arm completed no trials at all" — was written for a paired metric and is false for these two: they have one arm, and this gate cannot fire at zero loss. It was being rendered to operators on every degraded run. #379 gives the fixed-ideal metrics their own sentence.
- The comment above `_exclusion_remedy` cited `test_only_accuracy_is_ever_withheld` as proof a recall exclusion "cannot happen". **That test does not exist anywhere in the repo** — the citation was wrong when written, independently of this issue.

## Left open, deliberately

At 10% loss the report still prints `final-accuracy 90% **FAIL**`. That metric pairs against a measured control and is governed by `_unmeasured`'s own 20% share threshold — #352's design, not this issue's scope. It is arguably the same question one metric over, and worth its own issue rather than a silent widening of this one.
